test/Makefile.am

```diff
@@ -0,0 +1,5 @@
+check_PROGRAMS = UnitTest++/TestUnitTest++
```

Comment:
I'm not sure what this stuff is. I would honestly remove it until we get the view working, one thing at a time.